Example/YoshiExample/ViewController.swift

```diff
@@ -10,10 +10,20 @@ import UIKit
 
 internal final class ViewController: UIViewController {
 
-    @IBOutlet private weak var environment: UILabel!
+    @IBOutlet private weak var environment: UILabel! {
+        didSet {
+            environment.text = environmentName
```

Comment:
Noticed an issue here where if you switch your environment, close Yoshi and restart the app, the environment label is empty even though the Yoshi menu displays the correct environment.

Reply:
Fixed